hathor/simulator/simulator.py

```diff
@@ -229,16 +229,25 @@ def _run(self, interval: float, step: float, status_interval: float) -> Generato
 
     def run_until_complete(self,
                            max_interval: float,
+                           min_interval: float = 0.0,
```

Comment:
Maybe add an explanation for this parameter in the docstring. As far as I understood, it should advance the clock until `min_interval` even if all connections are complete. Is that correct?

Reply:
Great! Thanks!